components/motor/gpio/motor_encoder.go

```diff
@@ -412,7 +436,15 @@ func (m *EncodedMotor) computeRPM(pos, lastPos, now, lastTime int64) float64 {
 
 func (m *EncodedMotor) computeNewPowerPct(currentRPM, desiredRPM float64) float64 {
 	lastPowerPct := m.state.lastPowerPct
-	if math.Abs(currentRPM) <= 0.001 {
+
+	if desiredRPM*currentRPM < 0 {
+		// if desiredRPM and currentRPM are different signs, we're going the wrong direction
```

Comment:
can you add a to-do to test a negative SetPower Call interrupted by a GoFor in a the opposite direction? (I'm going to look at all these next week)

Reply:
i'm going to add some tests soon, don't need to add a TODO